jobclient/python/cookclient/instance.py

```diff
@@ -98,6 +98,8 @@ class Instance:
     :type status: cookclient.instance.Status
     :param preempted: If true, then this instance was preempted.
     :type preempted: bool
+    :param backfilled: If true, than this instance was backfilled.
+    :type backfilled: bool
```

Comment:
The `backfilled` field is deprecated -

https://github.com/twosigma/Cook/blob/5f44d7818f1a2b9132bf6cd11b7cf0586b2adb07/scheduler/src/cook/rest/api.clj#L1106

I would say we just leave it out of the python client.